api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -80,18 +81,9 @@ message HttpConnectionManager {
   google.protobuf.BoolValue add_user_agent = 6;
 
   message Tracing {
-    enum OperationName {
-      option (gogoproto.goproto_enum_prefix) = false;
```

Comment:
isn’t this backward incompatible change

Reply:
In the past we have allowed changes like this because there is no wire change, just a compile change. I'm not sure what the new policy is here. cc @htuch

Reply:
Yeah, we disallow these now. I suggest replicating and adding a TODO to clean this up in v3. Basically, we're going to start leaving around a trail of technical debt in the v2 major version APIs, we should tag this and then do a sweep in v3.